File: functions/notes.c
// Zoe jimenez, Function Notes C
#include <stdio.h>
int num;
char name[50], place[50], verb[50];
int add(int numOne, int numTwo){
    return numOne + numTwo;

}

const char* word(char type[50]){
    char choice[50];
    printf("Please give me a %s: \n", type);
    scanf("%s", choice);
    return choice;


}


int main(void){
    //printf("Please tell me a number: \n");
    //scanf("%d", num);
    //add(num,1);
    //add(8,1);
    add(72,1);
printf("%d",add);





    return 0;
}

File: functions/notes.py
# Zoe jimenez, function notes PYTHON

#functions hold actions to be reused
#number = int(input("Please tell me a number: \n"))
#def add(numOne, numTwo):
    #numOne = int(input("Please tell me a number: \n"))
    #numTwo = int(input("Please tell me another number: \n"))

 #   return numOne + numTwo
    #print(numOne+numTwo)


#add(number,267)
#print(add(8,354))
#addition = add(34,657)
#add(addition,965)
#add(3,284)

def values(type):
    return input(f"Please give me a {type}:")

name = values ("name")
place = values ("place")
verb = values ("verb (past tense)")

print(f"{name} was really fast getting to {place} by {verb} the whole way there.")